apps/admin-gui/src/app/shared/components/dialogs/create-relation-dialog/create-relation-dialog.component.ts

```diff
@@ -68,6 +78,25 @@ export class CreateRelationDialogComponent implements OnInit {
     this.dialogRef.close(false);
   }
 
+  getGroupsToInclude(voId: number): void {
```

Comment:
Change `this.data.group.voId` to `this.data.voId` on all occurences in this method. So there are not three different id references which is a little confusing.